features/step_definitions/basic_steps.rb

```diff
@@ -568,3 +568,11 @@
   expect(stderr).not_to include "Error"
   expect(status).to be_success
 end
+
+Then(/^I should have a page with a title: "([^"]*)"$/) do |title|
+  expect(page).to have_title 'Harrow Community Network'
+end
+
+Then(/^I should have a page with a description: "([^"]*)"$/) do |description|
+  expect(page).to have_description 'description'
```

Comment:
@lollypop27 same here you want to ensure that the content of `description` actually gets rendered on the page and not `'description'`